drop-boxes/register-mtb-data-dropbox/register-mtb-data-dropbox.py

```diff
@@ -392,7 +393,7 @@ def find_pbmc(in_file, transaction):
     if len(set(barcode)) > 1:
         raise mtbutils.MTBdropboxerror('More than one QBiC Barcode found in {}'.format(in_file))
     _, _, pbmc = getentityandpbmc(barcode[0], transaction)
```

Comment:
Here the refactor did not work? `barcode[0]` is still referenced here.

Reply:
it is fine there, because it is right after the regex search evaluation